.github/scripts/run.sh

```diff
@@ -47,9 +47,16 @@ export GOMP_CPU_AFFINITY="${CORE_LIST}"
 # done
 
 echo "Running check_lazy.py"
-python check_lazy.py --output_file ${DATA_DIR}/sweep.json
-# TODO(alanwaketan): Enable the below format conversion.
-# At the same time, we can workaround it by running the command by hands.
-# python check_lazy.py --json_to_csv ${DATA_DIR}/sweep.json --output_file ${DATA_DIR}/sweep.csv
+# The output is a file full of JSON objects but not legit .JSON.
+python check_lazy.py --output_file ${DATA_DIR}/sweep.out
+# Ignore the error for this command such that we can still have the json file
```

Comment:
what error are you ignoring? i never understood what was wrong with running the --json_to_csv before. but it looks like you got it working since I found a .csv in the artifacts you sent me.

Reply:
It's bogus. Let me remove them.